actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -3,6 +3,7 @@
 require "active_support/core_ext/array/access"
 require "active_support/core_ext/hash/keys"
 require "active_support/core_ext/string/output_safety"
+require "active_support/core_ext/object/blank"
```

Comment:
Rails automatically includes `active_support/core_ext/object/blank` and a [handful of other Active Support core extensions](https://github.com/rails/rails/blob/main/activesupport/lib/active_support/rails.rb), so no need to require here.